cli/runtime/version/zz_generated_plugin_runtime_version.go

```diff
@@ -0,0 +1,14 @@
+// Copyright 2021 VMware, Inc. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package version
+
+// IMPORTANT: Please !!DO NOT!!! change the content of this file manually
+// This file is auto-generated based on the version details provided during build time
+// and the values mentioned here are used to store the library version
+
+// All the variables are set during build time
+// Note: Please !!DO NOT!!! change the name or remove this variable
+var (
+	Version string = "v0.27.0-dev"
```

Comment:
this should now be "v0.28.0-dev" I think

Reply:
Is there a way to catch this thing as part of CI? I see we are ignoring this file as part of `verify-dirty.sh`, if we do not skip this file, will it create any issue?